ega-data-api-res/src/test/java/eu/elixir/ega/ebi/reencryptionmvc/SampleTestConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package eu.elixir.ega.ebi.reencryptionmvc;
+
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SampleTestConfiguration {
```

Comment:
Is this empty class needed?

Reply:
This class has the @SpringbootApplication that is used to determine where the app tests reside, and as the annotation was removed from the src/main...ReEncryptionMvcApplication.java this class helps spring to set up properly the tests